cmd/version/version.go

```diff
@@ -41,15 +41,13 @@ type Version struct {
 	GoArch             string `json:"goArch"`
 }
 
-func GetVersion() Version {
-	return Version{
-		kubeBuilderVersion,
-		kubernetesVendorVersion,
-		gitCommit,
-		buildDate,
-		goos,
-		goarch,
-	}
+var version = Version{
+	kubeBuilderVersion,
+	kubernetesVendorVersion,
+	gitCommit,
+	buildDate,
+	goos,
+	goarch,
 }
```

Comment:
Keep as it was before is clear and do more performative.

Reply:
Building a Version each time is not more performant. About the clearness, that could be discussed.

Reply:
I'd recommend you do like this [example](https://github.com/dev4devs-com/postgresql-operator/blob/master/pkg/config/backup.go). Note that has no need to use the memory space to store the version var and we can return it directly.

Reply:
Performance is not a concern here. Given that we don't have a good test coverage for this code and also `version` specific changes requires testing using build/release workflow (that is always a pain), I would recommend keeping changes to minimal here and may be making this method private `GetVersion` --> `getVersion` will be more reasonable change here.